src/server/console/logind_console_services.cpp

```diff
@@ -229,17 +232,45 @@ connect_to_system_bus(mir::GLibMainLoop& ml)
             {
                 auto error_msg = error ? error->message : "unknown error";
                 BOOST_THROW_EXCEPTION((
-                    std::runtime_error{
-                        "Failed to connect to DBus system bus: "s + error_msg}));
+                                          std::runtime_error{
+                                              "Failed to connect to DBus system bus: "s + error_msg}));
```

Comment:
There are a lot of spurious, whitespace changes to this file. A lot of them away from project style